model/project.py

```diff
@@ -18,7 +20,7 @@ def __init__(self, github_team_id, github_urls):
         """
         hfunc = hashlib.sha1()
         hfunc.update(bytes(github_urls[0], 'utf-8'))
-        hfunc.update(bytes(str(time.time()), 'utf-8'))
+        hfunc.update(bytes(str(uuid.uuid1()), 'utf-8'))
```

Comment:
What is `uuid1()` (i.e. why "1")? Also, does the UUID need to be converted to a string still? I imagine it returns a string already.

Reply:
uuid1 is the method that generates a UUID from a host ID, sequence number, and the current time. I could also use uuid4 which generates a random UUID. I also think it needs the str() as it gives type error without. Which method would be better? uuid1 would make a duplicate if it is from the same computer and the same time, while uuid4 would make a duplicate based on random luck(around 1/2^128)

Reply:
Well, 2^128 is many orders of magnitude larger than the age of the universe, so I don't think we really need to worry about that 😛 As long as it works, I'd say either is fine - I don't really know enough to say either way. Perhaps @cheukyin699 has an opinion?